ckanext/datajson/harvester_base.py

```diff
@@ -430,6 +431,17 @@ def import_stage(self, harvest_object):
                     'Import')
                 return None
 
+
+        # do title check here
+        # https://github.com/GSA/datagov-deploy/issues/953
+        title_to_check = self.make_package_name(dataset.get('title'), harvest_object.guid)
+        try:
+            name_validator(title_to_check, None)
+        except Invalid as e:
+            invalid_message = "title: %s. %s." % (dataset.get('title'), e.error)
+            self._save_object_error(invalid_message, harvest_object, 'Import')
```

Comment:
Can you include an INFO log line here for debugging?

Reply:
the function `_save_object_error` will print out a DEBUG message of `invalid_message`.